projects/kit/components/multi-select/multi-select.template.html

```diff
@@ -10,7 +10,7 @@
         automation-id="tui-multi-select__input"
         class="t-input"
         [nativeId]="nativeId"
-        [tuiTextfieldIcon]="icon"
+        [tuiTextfieldIcon]="arrow ? icon : ''"
```

Comment:
I have doubts about this screenshot.
Is the such new condition expected ?

![multiselect-icon](https://user-images.githubusercontent.com/35179038/228250549-be54afcb-8ae6-4eb0-ad63-344debf738ef.png)

Reply:
\+ this one

<img src="https://user-images.githubusercontent.com/35179038/228251064-4d1b47d3-2836-44e6-a997-f40591929a39.png">